cdap-ui/server/token.mjs

```diff
@@ -117,7 +117,7 @@ function getSecretFromCDAPConfig(cdapConfig, logger) {
  * 1. Get the following from cdap config (cdap-site + cdap-security)
  *    - Secret key, - secret
  *    - environment specific ID, - instancename
- * 2. Create a hash of (creation time + authToken + instance name)
+ * 2. Create a hash of (creation time + instance name)
```

Comment:
A collision between different users could occur if they hit this function at the same time. If there's a collision, will that weaken the overall token?

Reply:
Session token her is used only to validate the client connection. Anybody can get a sessiontoken from nodejs server from the `/sessionToken` endpoint. It is ok if there is a collision as it only validates the client and not share any sensitive data.